packages/core-sdk/src/resources/group.ts

```diff
@@ -147,7 +148,9 @@ export class GroupClient {
         spgNftContract: getAddress(spgNftContract, "request.spgNftContract"),
         recipient:
           (recipient && getAddress(recipient, "request.recipient")) || this.wallet.account!.address,
-        maxAllowedRewardShare: BigInt(getRevenueShare(request.maxAllowedRewardShare)),
+        maxAllowedRewardShare: BigInt(
+          getRevenueShare(request.maxAllowedRewardShare, RevShareType.MAX_ALLOWED_REWARD_SHARE),
```

Comment:
Please don't mix changes that is not part of the rc1 release plans with changes that is included. Combining them does not allow us to not include these in the next release if its not ready.
This is fine for now, but in the future, this needs to be separate PR.

Reply:
Please ensure I've reviewed any new items you've added to rc.1. I did not know this was added to this release. 
As for if this needs a separate PR, yes it does. This change is not related to any other change in this PR. Each PR should do one thing or a few changes of a similar issue. Small PRs are normal and very much fine.